src/test/java/org/mule/extension/ftp/FtpFileAttributesTestCase.java

```diff
@@ -33,7 +35,7 @@
 public class FtpFileAttributesTestCase {
 
   private static final String EXPECTED_FILENAME = "hello.txt";
-  private static final String EXPECTED_PATH = "/root/" + EXPECTED_FILENAME;
+  private static final String EXPECTED_PATH = lineSeparator() + "root" + lineSeparator() + EXPECTED_FILENAME;
```

Comment:
Why lineSeparator?

Reply:
Because when you want to get the path, the ´FtpUtils´ class is used with the ´normalizePath()´ method and this is where you decide if you use a Windows or Unix separator.